plugins/pixart-rf/fu-pxi-firmware.c

```diff
@@ -234,3 +278,15 @@ fu_pxi_firmware_new(void)
 {
 	return FU_FIRMWARE(g_object_new(FU_TYPE_PXI_FIRMWARE, NULL));
 }
+
+void
+fu_pxi_firmware_set_is_hpac_flag(FuPxiFirmware *self, gboolean flag)
+{
+	self->is_hpac_flag = flag;
+}
```

Comment:
with detecting the magic, you shouldn't need this anymore.

Reply:
solved in 269968b90c62b0300ff7b0ef900798a43124ff41